.github/workflows/check-links.yaml

```diff
@@ -0,0 +1,44 @@
+name: check-links
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - 'cmd/builder/**'
+  pull_request:
+    paths-ignore:
+      - 'cmd/builder/**'
```

Comment:
Contrib doesn't have `cmd/builder`.  Do we have an equivalent we need to ignore or can we remove this?

Reply:
Removed